srcpkgs/rpi-base/template

```diff
@@ -3,9 +3,9 @@ pkgname=rpi-base
 version=2.6
 revision=1
 archs="armv6l* armv7l* aarch64*"
-depends="virtual?ntp-daemon rpi-firmware rpi-kernel"
+depends="virtual?ntp-daemon rpi-firmware"
```

Comment:
I still think current users shouldn't have to change anything about their current systems to get things working. The amount of information contained in this package is small enough that a `rpi4-base` package wouldn't be undue maintenance in almost any way.

Reply:
What current systems? Void does not currently support RPI4 without a lot of manual work.

Reply:
Current users of the `rpi-base` packages, so rpi zero, 1, 2 and 3 owners.

Reply:
Those are technically different architectures. I wouldn't think it wise to try and run an install targeting any of those on any other device.

Reply:
I'm not sure I get what you mean.

My point is that `rpi-base` currently works and pulls all dependencies for current rpi users. With this change, they will also have to download the kernel for their specific board. It would be simpler for them, without much more work on our side, to have a `rpi4-base` package for rpi4 users (it could even be a subpackage here), while current users don't get an orphaned kernel package or have to remember to install new systems with `rpi-base` *and* `rpi$something-kernel`.

Reply:
I don't like the idea of a separate `rpi4-base` because the distinction is arbitrary. The problem compounds when there is a Raspberry Pi 5 and so on. Suddenly every new hardware release that requires a different kernel gets paired with a new `-base` just to pull in that kernel.

The only difference needed to move from rpi3 to rpi4 is the kernel, so let's decouple the kernel from the base and keep things consistent.

Reply:
that was my point, yet i didn't come up with a solution that doesn't require user intervention for existing install (1rpi{0,1,2,3}1)

Reply:
Having per-model bases seems better than playing "one of these things is not like the other" with `rpi-base` and `rpi4-base`. I don't think there's a lot of added value in making the `-base` depend on the kernel, but if there is agreement with the position of @ericonr, might as well split them all if we're splitting one.

Reply:
For what it's worth, rpi0 and rpi1 would be the same package.